openadapt/config.py

```diff
@@ -1,45 +1,101 @@
+from dotenv import find_dotenv, load_dotenv
+from loguru import logger
 import multiprocessing
 import os
 import pathlib
 
-from dotenv import load_dotenv
-from loguru import logger
+ROOT_DIRPATH = pathlib.Path(__file__).parent.parent.resolve()
+ZIPPED_RECORDING_FOLDER_PATH = ROOT_DIRPATH / "data" / "zipped"
+
+ENV_FILE_PATH = (ROOT_DIRPATH / ".env").resolve()
+logger.info(f"{ENV_FILE_PATH=}")
+dotenv_file = find_dotenv()
+load_dotenv(dotenv_file)
+
+
+def read_env_file(file_path):
```

Comment:
I'm having trouble understanding the purpose of this function, could we just use os.getenv (or similar) instead?

Reply:
Oh I see, for the load_dotenv(), it might be dependant on the location of your .env file (looks in same directory as the entry file rather than parent)

Reply:
For the error I'm getting, it's on the export - what should I do? Do I need more than one recording?

Reply:
You shouldn't have to, but yeah try making another recording, and make sure your .env file DB_FNAME = 'openadapt.db', or whatever db you want to export from

Reply:
I am still confused by this file.

How about e.g.:

```
ROOT_DIR_PATH = pathlib.Path(__file__).parent.resolve()
ENV_FILE_PATH = (ROOT_DIR_PATH / ".." / ".env").resolve()
load_dotenv(ENV_FILE_PATH)
```

Reply:
I'm having issues with `os.getenv` not actually loading the env values, which is why in the function I directly read off the .env file.

Reply:
@Mustaballer I believe we discussed you would remove this, is that right?

Reply:
Yes, I will be removing the current function :)

Reply:
resolved in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/981933027f3299d88783b5d88d57c24ac3e620c8